src/SeleniumLibrary/__init__.py

```diff
@@ -389,6 +389,31 @@ def browser(self):
             raise RuntimeError('No browser is open')
         return self._browsers.current
 
+    def find_element(self, locator, parent=None):
+        """Find element matching ``locator``.
+
+        This method and :meth:`find_elements` form the recommended
+        public API for external tools to get elements via SeleniumLibrary.
+
+        :param locator: Locator to use when searching the element.
+            See library documentation for the supported locator syntax.
+        :param parent: Optional parent ``WebElememt`` to search child elements
+            from. By default search starts from the root using ``WebDriver``.
+        :rtype: selenium.webdriver.remote.webelement.WebElement
+        :raises SeleniumLibrary.errors.ElementNotFound: If element not found.
+        """
+        return self._element_finder.find(locator, parent=parent)
+
+    def find_elements(self, locator, parent=None):
```

Comment:
Should this have similar sphinx style parameter definitions as find_element in above?

Reply:
It wouldn't hurt. More important change would be adding return type information, but it should mention that return type is a list of `WebElement` objects, not just that the type is a list.

Reply:
You had some other syntax changes to type info. Can you take care of all of them at the same time?

Reply:
Yes I can. I will merge this PR and write separate PR for fixing it.